.github/workflows/lint.yml

```
@@ -71,15 +71,17 @@ jobs:
         working-directory: build
         run: |
           export CONAN_LLVM_GIT_CACHE="${{ runner.temp }}/llvm-project"
-          conan install .. -pr:h default -pr:b default
-          if [ $? -ne 0 ]
+          build_requires=$(conan info .. -pr:h default -pr:b default --build=missing 2>/dev/null)
+          if [ build_requires ]
           then
-              cache_hit=false
+              all_in_cache=true
           else
-              cache_hit=true
+              all_in_cache=false
           fi
-          echo "cache_hit=$cache_hit" >> $GITHUB_OUTPUT
-          echo "Cache was hit: $cache_hit"
+          echo "all_in_cache=$all_in_cache" >> $GITHUB_OUTPUT
+          echo "Conan build requires:"
+          echo $build_requires
```

To be consistent with the next line, you might consider
```suggestion
          echo "Conan build requires: $build_requires"
```